sklearn/metrics/pairwise.py

```diff
@@ -164,44 +197,32 @@ def euclidean_distances(X, Y=None, Y_norm_squared=None, squared=False,
     """
     Considering the rows of X (and Y=X) as vectors, compute the
     distance matrix between each pair of vectors.
-
     For efficiency reasons, the euclidean distance between a pair of row
     vector x and y is computed as::
-
```

Comment:
Haha good one. But I think I misunderstood the comment -- @jnothman what do you mean by "remind a whole lot of blank lines"? Did you mean "removed"?